README.md

```diff
@@ -140,13 +140,25 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :revoked_at, default: Time.now
+    t.datetime :expiration_time
   end
   add_index :jwt_blacklist, :jti
 end
 ```
-
 For performance reasons, it is better if the `jti` column is an index.
 
+Note, if you used the blacklist strategy in a previous version you may not have the fields *revoked_at* and *expiration_time.* If not run the following migration:
+```
+class AddRevokedAtAndExpirationTimeToJWTBlacklist < ActiveRecord::Migration
+  def change
+    add_column :JWTblacklist, :revoked_at, :datetime, default: Time.now
+    add_column :JWTblacklist, :expiration_time, :datetime
```

Comment:
You are right. @waiting-for-dev , I don't know if you noticed this but the test fixture goes by "blacklist" as oppose to "jwt_blacklist". Just an small inconsistency I noticed between readme and tests.